src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
+use Closure;
```

Comment:
to be removed, we use `\Closure` inline instead, always

Reply:
Ok, thanks. (therefore you may have a look at `src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php`)